### DISCUSSION TITLE
Does rtlamr scan an entire frequency range or just one frequency?

### DISCUSSION THREAD

**Discussion by sofakng** [bemasher/rtlamr#261] [General]
I have a couple of meters that I'm trying to read (ie. electric, gas, water) but it seems like rtlamr (by default) just monitors a single default frequency.

Do I need to adjust this to scan a range of frequencies?  (do I need to find my meters frequency first using another method?)

**Comment by bemasher** [comment]
I'm going to translate frequency in your post to the word "channel."

There are a lot of meters operating in the 915Mhz ISM band, with different behaviors. Some transmit on just one channel, some on multiple. But you often don't get to know that in advance and listen accordingly.

Rtlamr is designed with promiscuity in mind and can detect signals anywhere within the full bandwidth of the radio, centered about the frequency it is parked on. In most cases, especially for meters that hop channels, you wont need to adjust the center frequency unless you know you have a meter that uses only a single channel outside the center frequency +/- half the sample rate.
